modules/tinymce/src/core/test/ts/browser/FontSelectTest.ts

```diff
@@ -80,6 +80,32 @@ UnitTest.asynctest('browser.tinymce.core.FontSelectTest', function (success, fai
         sAssertSelectBoxDisplayValue(editor, 'Fonts', 'Times')
       ])),
 
+      // https://websemantics.uk/articles/font-size-conversion/
```

Comment:
Should we move (or copy) this comment over to the declaration of `keywordFontSizes` in FontsizeSelect.ts?

Reply:
It was already mentioned on the other mapping object

https://github.com/tinymce/tinymce/blob/f283cfa7b49272e7518a851b912804ea58a89601/modules/tinymce/src/themes/silver/main/ts/ui/core/complex/FontsizeSelect.ts#L19

Reply:
Awesome